src/components/20-molecules/datepicker/ui.test.js

```diff
@@ -687,16 +708,17 @@ fixture('Datepicker as inputfield with 196px width and 10px height').page(
   `${host}/iframe.html?id=components-molecules-datepicker--datepicker&knob-inputfield=true&knob-locale=de-CH&knob-year=2020&knob-month=4&knob-day=22&knob-disabled=&knob-autofocus=&knob-checkMark=&knob-label=&knob-monthtitle=Choose Month&knob-yeartitle=Choose Year&knob-invaliddatetext=Invalid date&knob-placeholder=Please select a date&knob-width=196&knob-height=10`
 );
 
-test('should have a minimum width', async t => {
+test('should have no minimum width', async t => {
   const datepicker = await Selector(() =>
     document.querySelector('axa-datepicker')
   );
   const datepickerInputWrap = await datepicker.find(
     '.m-datepicker__input-wrap'
   );
   await t.expect(datepicker.clientWidth).eql(196); // component has no min-width
-  await t.expect(datepickerInputWrap.clientWidth).eql(197); // input wrapper has min-width
+  await t.expect(datepickerInputWrap.clientWidth).eql(196); // input wrapper has no min-width
```

Comment:
What is the reason for this 1px difference? (197 -> 196)

Reply:
The min-width. It was 197px. And now without that css rule we can have 196 (or lower) px width.